pkg/cmd/clustertask/testdata/Test_ClusterTask_Start-Dry_run_with_no_output.golden

```diff
@@ -6,13 +6,12 @@ metadata:
   labels:
     key: value
 spec:
-  inputs:
-    resources:
+  resources:
```

Comment:
this is wrong spec for v1alpha1, need to also handle the dryrun like create

Reply:
Thanks for this one. I will update the PR.
We will need to handle this in task start as well.